pyrasterframes/python/pyrasterframes/types.py

```diff
@@ -13,7 +13,7 @@ class here provides the PyRasterFrames entry point.
 from pyspark.ml.util import JavaMLReadable, JavaMLWritable
 from pyrasterframes.context import _checked_context
 
-__all__ = ['RFContext', 'RasterFrame', 'TileUDT', 'TileExploder']
+__all__ = ['RFContext', 'RasterFrame', 'TileUDT', 'GeometryUDT', 'TileExploder']
```

Comment:
Does `NoDataFilter` need to be here?
Why is adding `GeometryUDT` a part of this PR?.... just forgotten from before?